src/integrations/prefect-gcp/prefect_gcp/models/cloud_run_v2.py

```diff
@@ -390,5 +388,5 @@ def get(
         )
 
 
-class CloudRunJobV2Result(InfrastructureResult):
+class CloudRunJobV2Result:
```

Comment:
I think this class can also be removed.

Reply:
It is a type hint for return type for `CloudRunWorkerV2`. Removing the type hints ok?

Reply:
I'm just merging this for now to unblock another PR but happy to remove these after if so!